src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -224,37 +222,22 @@ async function stopMinikube(context: Context): Promise<void> {
     });
 }
 
-function getStatusFormatString(currentVersion: string): string {
-    if (semver.lt(currentVersion, "0.31.0")) {
-        // minikube < 0.31.0 uses this format
-        return `"{{.MinikubeStatus}}","{{.ClusterStatus}}","{{.KubeconfigStatus}}"`;
-    } else if (semver.lt(currentVersion, "1.0.0")) {
-        // minikube < 1.0.0 uses this format
-        return `"{{.Host}}",{ "kubelet": "{{.Kubelet}}", "apiserver": "{{.ApiServer}}" },"{{.Kubeconfig}}"`;
-    }
-
-    // minikube >= 1.0.0 uses this format
-    return `"{{.Host}}",{ "kubelet": "{{.Kubelet}}", "apiserver": "{{.APIServer}}" },"{{.Kubeconfig}}"`;
-}
-
 async function minikubeStatus(context: Context): Promise<MinikubeInfo> {
     if (!await checkPresent(context, CheckPresentMode.Silent)) {
-        throw new Error('minikube executable could not be found!');
+        throw new Error('Minikube executable could not be found!');
     }
 
-    const statusFormatString = getStatusFormatString((await getVersionInfo(context)).currentVersion);
+    const result = await context.shell.exec(`"${context.binPath}" status`);
 
-    const result = await context.shell.exec(
-        `"${context.binPath}" status --format '[${statusFormatString}]'`);
     if (result && result.stderr.length === 0) {
-        const obj = JSON.parse(result.stdout);
+        const hostStatus = result.stdout.split('\n')[0].split(': ')[1].toLowerCase();
```

Comment:
Personally, I do not like the way i did the parsing here. Maybe there is a more safe and elegant way...